ballerina/tests/resources/sql-scripts/init.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE test.Employee (
     firstName VARCHAR(30),
     lastName VARCHAR(30),
     birthDate DATE,
-    gender CHAR(1),
+	gender ENUM('MALE', 'FEMALE') NOT NULL,
```

Comment:
```suggestion
    gender ENUM('MALE', 'FEMALE') NOT NULL,
```

Reply:
Fixed in [ea8ebbe](https://github.com/ballerina-platform/module-ballerina-persist/pull/199/commits/ea8ebbed23186a93f394117d9449e7faaadb9385)